jurisdiction/README.md

```diff
@@ -0,0 +1,156 @@
+# Mobility Data Specification: Jurisdiction **[Beta](/general-information.md#beta-features):**
```

Comment:
For the beta label, you can remove this from the title, and instead add a section on the page.  See [metrics](https://github.com/openmobilityfoundation/mobility-data-specification/tree/feature-metrics/metrics#beta-feature) for an example.